docker-compose-infra.yml

```diff
@@ -37,6 +35,24 @@ services:
         reservations:
           cpus: '0.05'
           memory: 50M
+  test-things-auth:
+    image: thomseddon/traefik-forward-auth:2-arm64
```

Comment:
Will need to check if we can use the arm image

Reply:
This repo for forward auth is not very well managed, even though this says arm64 all the images in Docker Hub are amd64. I can rename this.